dts/bindings/flash_controller/renesas,ra-flash-hp-controller.yaml

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2024 Renesas Electronics Corporation
```

Comment:
Just a question.

Is there a good way to handle these FSP configurations in devicetree?

https://github.com/zephyrproject-rtos/hal_renesas/blob/32c029e30141bff85fb18f4c86103334133ee6c5/zephyr/ra/ra_cfg/fsp_cfg/r_flash_hp_cfg.h#L14-L15

Reply:
Hi @soburi ,
Thanks for your comment. I added properties in dts/bindings/mtd/renesas,ra-nv-flash.yaml and opened a PR for hal_renesas at https://github.com/zephyrproject-rtos/hal_renesas/pull/33 to handle this.